File: BECore/app/model/ProjectWorkshop_AttachDetail.js
/// <summary>
/// Model EXTJS ProjectWorkshop_AttachDetail.
/// </summary>
Ext.define('QLDT.model.ProjectWorkshop_AttachDetail', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectWorkshop_AttachDetailID',
    idType: 'Guid',
    fields: [
        { name: "ProjectWorkshop_AttachDetailID", type: "string" },
		{ name: "ProjectWorkshopID", type: "string" },
        { name: "Description", type: "string" },
		{ name: "FileName", type: "string" },
		{ name: "FileType", type: "string" },
		{ name: "FileSize", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: "IsTemp", type: "boolean" },
    ]
});

File: MT.Model/Business/ProjectWorkshop_AttachDetail.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ProjectWorkshop_AttachDetail:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ProjectWorkshop_AttachDetail:EntityBase.
    /// </summary>
    public class ProjectWorkshop_AttachDetail : EntityBase
    {
        [Key]
        public Guid ProjectWorkshop_AttachDetailID { get; set; }
        public Guid ProjectWorkshopID { get; set; }
        public string FileName { get; set; }
        public string FileType { get; set; }
        public double FileSize { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
        public Guid? FileResourceID { get; set; }
        public string Description { get; set; }
    }
    #endregion
}

